react/features/security/components/security-dialog/web/PasswordSection.tsx

```diff
@@ -106,24 +103,16 @@ function PasswordSection({
      * @private
      * @returns {void}
      */
-    function onTogglePasswordEditState() {
-        if (typeof APP === 'undefined' || !buttonsWithNotifyClick?.length) {
+    const onTogglePasswordEditState = useCallback(() => {
+        if (typeof APP === 'undefined' || !buttonsWithNotifyClick?.size) {
             setPasswordEditEnabled(!passwordEditEnabled);
 
             return;
         }
 
-        let notifyMode;
-        const notify = buttonsWithNotifyClick.find(
-            (btn: string | INotifyClick) =>
-                (typeof btn === 'string' && btn === KEY)
-                || (typeof btn === 'object' && btn.key === KEY)
-        );
+        const notifyMode = buttonsWithNotifyClick?.get(KEY);
 
-        if (notify) {
-            notifyMode = typeof notify === 'string' || notify.preventExecution
-                ? NOTIFY_CLICK_MODE.PREVENT_AND_NOTIFY
-                : NOTIFY_CLICK_MODE.ONLY_NOTIFY;
+        if (notifyMode) {
```

Comment:
how do we know here that notifyMode is ` NOTIFY_CLICK_MODE.PREVENT_AND_NOTIFY` ?